### DISCUSSION TITLE
inference time

### DISCUSSION THREAD

**Discussion by gubei528** [obss/sahi#745] [Q&A] [answered]
A 640*640 image, using yolov5 to infer about 6-7ms. A 1920*1680 image, using SAHI slices, slice_height640 slice_width 640, overlap_height_ratio0.2, cut into 8 parts, why does it take 170ms?

**Comment by fcakyon** [comment, marked as answer]
Yolov5 resizes large large image to 640 before running. sahi crops 640 slice from image and gives this slice as 640 to model.

If your image 1280 size, then sahi applies 4x enhancement which may be slower..

**Reply by gubei528** [reply]
The inference time for this image is too long. A 640 image is 7ms, and the 1920*1080 image is divided into 8 parts. The inference time is too much time difference between the 640 image multiplied by 8.
